src/components/LocationSearch.js

```diff
@@ -5,12 +5,7 @@ import { Autocomplete } from "@react-google-maps/api";
 import CancelIcon from "@mui/icons-material/Cancel";
 import "./LocationSearch.css";
 import { IconButton, InputAdornment } from "@mui/material";
-const LocationSearch = ({
-  type,
-  initialLocation,
-  onLocationChange,
-  onCancelDisabled,
-}) => {
+const LocationSearch = ({ type, initialLocation, onLocationChange }) => {
   const [location, setLocation] = useState(initialLocation);
```

Comment:
location is needed for the representation of the predicted result of google autocomplete API, which is different from the response of getPlaceApi.